src/utils.h

```diff
@@ -327,6 +331,15 @@ namespace Utils {
 	 */
 	bool StringIsAscii(std::string& s);
 
+	/**
+	 * Formats a date.
+	 *
+	 * @param tm Time structure
+	 * @param format Format string (See strftime)
+	 * @return formatted date
+	 */
+	std::string FormatDate(const std::tm* tm, StringView format);
+
```

Comment:
Not sure if it is worth, but https://github.com/EasyRPG/Player/blob/10a3bbe2fcc32196765e7e1a034e5248cdf9736b/src/output.cpp#L83-L92 could use this too.

Reply:
yeah I will change it.
btw fmtlib also has a nice time-formater but this needs fmtlib 6.0 and you know how far behind some distributions are :/